lib/tapioca/compilers/dsl/protobuf.rb

```diff
@@ -0,0 +1,95 @@
+# typed: strict
+# frozen_string_literal: true
+require "parlour"
+
+begin
+  require "google/protobuf"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      class Protobuf < Base
+        extend T::Sig
+
+        sig do
+          override.params(
+            root: Parlour::RbiGenerator::Namespace,
+            constant: T.class_of(Google::Protobuf::MessageExts)
+          ).void
+        end
+        def decorate(root, constant)
+          descriptor = T.let(T.unsafe(constant).descriptor, Google::Protobuf::Descriptor)
+          return if descriptor.to_a.empty?
```

Comment:
Nitpick: Wanted to make sure that `to_a` is a good way to check for emptiness. There might be an alternative that isn't as costly to allocate a (potentially) large array?

Reply:
Very good point Kaan! 
 `constant.descriptor` doesn't accept `empty?`, so this is one suggestion that we could come up with.
any other suggestion is appreciated ❤️

Reply:
I found `any?` method locally or `none?`. I didn't go through the source just inspected its method at runtime so take those with a grain of salt 😀

Reply:
Yeah, we can use `any?` here to mean not-`empty?` since we don't expect the contents to have falsy values. However, be careful that in general `any?` is not strictly the same an not-`empty?` since:
```ruby
[false, nil].any? #=> false
```